subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/AbstractDependencyReportTask.java

```diff
@@ -141,5 +146,6 @@ private Set<Configuration> getNonDeprecatedTaskConfigurations() {
     }
 
     @Internal
+    @ToBeKeptEagerProperty(because = "ConfigurationContainer is injected type")
     public abstract ConfigurationContainer getTaskConfigurations();
```

Comment:
❗ Should not need annotation as a `DOC`.

Reply:
Done